api/admin/controller.py

```diff
@@ -462,6 +555,74 @@ def edit(self, identifier_type, identifier):
             staff_edition.series_position = new_series_position
             changed = True
 
+        new_medium = flask.request.form.get("medium")
+        if new_medium:
+            if new_medium not in Edition.medium_to_additional_type.keys():
+                self._db.rollback()
+                return UNKNOWN_MEDIUM.detailed(
+                    _("Medium %(medium)s is not one of the known media.",
+                      medium=new_medium))
+            staff_edition.medium = new_medium
+            changed = True
```

Comment:
If you change a title's medium, it seems like there's currently a risk that you'll remove it from the interface with no way to see it or change it back.